src/org/usfirst/frc4904/robot/humaninterface/drivers/DefaultDriver.java

```diff
@@ -11,7 +15,17 @@ public DefaultDriver() {
 	}
 	
 	@Override
-	public void bindCommands() {}
+	public void bindCommands() {
+		Command normalDrive = new ChassisMove(RobotMap.Component.chassis, this);
+		RobotMap.HumanInput.Driver.xbox.dPad.up.whenPressed(new ChassisTurnAbsolute(RobotMap.Component.chassis, 180, RobotMap.Component.navx, RobotMap.Component.chassisMC));
```

Comment:
While these angles are the most logical for a 90* system, I think it is worth taking into account the fact that many of the targets on the field we are interested in (boiler, gear peg) are at 45* or 120*. If you can think of a nice way to handle this, that would be really nice. This may require another command to determine what the most reasonable angle is before starting this command.